### PR TITLE
Our structure

### PR DESCRIPTION
**Pull request by motaz99**

### CODE REVIEW

File: src/component/ShortBreakTimer/index.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+const ShortBreakTimer = () => {
+    return (
+        <div>
+            
+        </div>
+    )
+}
+
+export default ShortBreakTimer
```
Comment (mujz) [inline src/component/ShortBreakTimer/index.js:11]: For now you can leave this the way it is. But in the future, you should not have separate components for the short break timer, the long break timer, and the Pomodoro timer. They should all be one timer component and just change the number displayed.

**Review by mujz — APPROVED** [review]

File: src/component/ShortBreakTimer/index.js (hunk 1 above)
Comment (motaz99) [reply src/component/ShortBreakTimer/index.js:11]: despair

**Comment by murtadha** [comment]
despair is punishment for the undead

File: src/component/Timer/index.js [hunk 2]
```diff
@@ -1,10 +1,9 @@
 import React from 'react'
-import TimerLogic from './TimerLogic'
 
 const Timer = () => {
     return (
         <div>
-            <TimerLogic/>
+            <button>Despair</button>
```
Comment (murtadha) [inline src/component/Timer/index.js:6]: .... is punishment for the undead

**Pull request merged**